de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/session/SarosSessionManager.java

```diff
@@ -259,17 +260,16 @@ public void startSession(
             referencePointManager = session
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this useless assignment to local variable "referencePointManager". ([squid:S1854](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS1854))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=273&issues=AWaWrfZtsSPV6ZXI_B89&open=AWaWrfZtsSPV6ZXI_B89)

[ISSUE_KEY]: AWaWrfZtsSPV6ZXI_B89